snap/snapcraft.yaml

```diff
@@ -5,12 +5,76 @@ license: BSD-3-Clause
 grade: stable
 confinement: strict
 contact: solutions-engineering@lists.canonical.com
-summary: DERP Relay Server for Tailscale
+issues: https://github.com/canonical/derper-snap/issues
+summary: DERP Server for Tailscale
 description: |
-  Self hosted DERP Relay servers for Tailscale networks.
-  This snap provides a simple way to run your own DERP servers to route your tailscale traffic through your own relay servers.
+  Derper is a self hosted DERP (Designated Encrypted Relay for Packets) server, designed to be used with Tailscale.
+  This snap provides a simple and secure way to run your own DERP servers to route your tailscale traffic through your own relay servers.
+
+  Please refer to the upstream documentation at https://tailscale.com/kb/1118/custom-derp-servers and https://tailscale.com/kb/1232/derp-servers for more information.
+
+  **Usage**
+
+  Install the snap:
+
+  ```bash
+  sudo snap install derper
+  ```
+
+  This installs the DERP server and starts it automatically.
+  You will likely need to configure the service:
+
+  **Configuration**
+
+  You can modify configuration options with `sudo snap set`. The following configuration options are currently supported:
+
+  * `hostname`: LetsEncrypt host name, if addr's (`a`) port is :443. (default if unset: "derp.tailscale.com")
+  * `a`: server HTTP/HTTPS listen address, in form ":port", "ip:port", or for IPv6 "[ip]:port". If the IP is omitted, it defaults to all interfaces. Serves HTTPS if the port is 443 and/or certmode is manual (default certmode is "letsencrypt"), otherwise HTTP. (default if unset: ":443")
+  * `stun-port`: The UDP port on which to serve STUN. The listener is bound to the same IP (if any) as specified in the `a` option. (default if unset: "3478")
+
+  The config options correspond to the similarly named command line flags for `derper`.
+
+  Options that don't have support to be configured through the snap will be left on their default.
+  See the `derper` command source code at https://github.com/tailscale/tailscale/tree/main/cmd/derper for more technical information on options.
+
+  Example of setting an option:
+
+  ```bash
```

Comment:
There are no code blocks in their markdown; please refer to one of my docs for either dmg, smart-exporter or charmed-openstack-exporter.  As I said before, the syntax is really really lean. You can test your docs by going to the listing and temporarily inserting it in the listing's description(preview won't work, so you will have to save it first)

Reply:
Ah of course, indenting rather than backticks :+1: